### DISCUSSION TITLE
Need to retain the old evaluation history results, including input and all ground truth

### DISCUSSION THREAD

**Discussion by serlina** [langfuse/langfuse#3252] [Ideas]
### Describe the feature or potential improvement

When using a dataset for evaluation, it is often necessary to update the dataset items, especially the ground truth. After the update, the old ground truth is no longer visible. As a result, the evaluation history becomes difficult to interpret because the standard answers have been modified. Previously, the score was 80, but now, due to the changes in the answers, it has become 95. However, for the evaluation with a score of 80, we are unable to see the old standard answers, making it difficult to explain the score reason.

**Comment by marcklingen** [comment]
Version control of datasets was raised a couple of times as an interesting extension of datasets.

I have seen some teams work around this by copying the dataset whenever they make larger changes. Would this work for you as well?

**Reply by serlina** [reply]
sure, I think using another dataset would be fine if we have larger changes. But if we're in developing phase, we may need to do smaller adjust on dataset, then the the version control of dataset items would be real requirement.

**Reply by AlbertInRC** [reply]
Can we pls add this feature? @marcklingen

**Reply by AlbertInRC** [reply]
We also need this feature to compare the results become better or not.
